ui/js/component/fileCard/view.jsx

```diff
@@ -94,12 +95,11 @@ class FileCard extends React.PureComponent {
                 <TruncatedText lines={1}>{title}</TruncatedText>
               </div>
               <div className="card__subtitle">
-                <span style={{ float: "right" }}>
+                <IconSet>
                   <FilePrice uri={uri} />
                   {isRewardContent && <span>{" "}<IconFeatured /></span>}
```

Comment:
`FilePrice` can be inside a `IconSet`?

Reply:
`IconSet` represents an info panel which is primarily designed to show icons but can also have other info (like a system tray which has a load of icons + the time on one side). I couldn't think of a better name for it. `IndicatorPanel`? I dunno.

We could also make the price completely separate from the icon set, although that will probably lead to additional presentational divs or spans to group them together wherever both a price and icons are being shown.

Reply:
Why does this exist at all actually, if it doesn't have styles? (other than the float, which is really a property of the card)

Reply:
I suppose it doesn't need to exist anymore. Unless we wanted to change it to a fancy component that displays the various indicators via props. I'll switch it back to a div, with the `float: right` handled by card level styles.